### ISSUE TITLE
New release doesn't launch

### ISSUE THREAD

**Issue by halvabner** [zacharee/SamloaderKotlin#113]
```

Exception in thread "main" java.lang.NoClassDefFoundError: java/lang/management/ManagementFactory
	at com.soywiz.kmem.internal.CurrentJvmKt$currentIsDebug$2.invoke(currentJvm.kt:38)
	at com.soywiz.kmem.internal.CurrentJvmKt$currentIsDebug$2.invoke(currentJvm.kt:37)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at com.soywiz.kmem.internal.CurrentJvmKt.getCurrentIsDebug(currentJvm.kt:37)
	at com.soywiz.kmem.internal.CurrentCommonKt.getCurrentBuildVariant(currentCommon.kt:15)
	at com.soywiz.kmem.internal.CurrentJvmKt.<clinit>(currentJvm.kt:44)
	at com.soywiz.kmem.Os$Companion.getCURRENT(Os.kt:22)
	at com.soywiz.kmem.Platform$Companion.getOs(Platform.kt:38)
	at com.soywiz.kmem.PlatformKt.isWindows(Platform.kt:72)
	at MainKt.main(main.kt:34)
	at MainKt.main(main.kt)
Caused by: java.lang.ClassNotFoundException: java.lang.management.ManagementFactory
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	... 11 more

```
	
I get this when I try to launch it. Running on Arch Linux

`Linux 6.5.3-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 13 Sep 2023 08:37:40 +0000 x86_64 GNU/Linux`

**Comment by pevik** [comment]
I also experienced it on Linux distro. Previous version launched (but suffered https://github.com/zacharee/SamloaderKotlin/pull/109).

I would bisect, but `./gradlew createDistributable` failed for me (I suspect some config issue, but I haven't tried 1.42.1, maybe that's how regression demostrates).

**Comment by pevik** [comment]
Error log prints:
```
Error: --module-path is not specified and this runtime image does not contain jmods directory.
Usage: jlink <options> --module-path <modulepath> --add-modules <module>[,<module>...]
Use --help for a list of possible options
```

**Comment by pevik** [comment]
Error log prints:
```
Error: --module-path is not specified and this runtime image does not contain jmods directory.
Usage: jlink <options> --module-path <modulepath> --add-modules <module>[,<module>...]
Use --help for a list of possible options
```

**Comment by BaigHack3rss** [comment]
I got it working. Clone the repo and run  `./gradlew run` in the project's root directory

**Comment by zacharee** [comment]
This should be fixed in 1.14.3.
